implementations/micrometer-registry-stackdriver/src/main/java/io/micrometer/stackdriver/StackdriverHistogramUtil.java

```diff
@@ -51,4 +51,12 @@ static class StackdriverFixedBoundaryHistogram extends TimeWindowFixedBoundaryHi
 
     }
 
+    static class StackdriverInfinityBucketHistogram extends TimeWindowFixedBoundaryHistogram {
+
+        StackdriverInfinityBucketHistogram(Clock clock, DistributionStatisticConfig config) {
+            super(clock, config, false, false, true);
```

Comment:
To explain the parameters here: 

* supportsAggregablePercentiles = false, so there will be no buckets from the percentile histogram buckets
* isCumulativeBucketCounts = false, like the other histograms
* includeInfinityBucket = true, to include the one bucket we want

If there were SLOs configured, those would be added too, but as stated above, we know this doesn't get called unless there are no buckets and no client percentiles.